src/sentry/integrations/slack/message_builder/notifications/daily_summary.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+from sentry_relay.processing import parse_release
+
+from sentry.integrations.message_builder import build_attachment_title, get_title_link
+from sentry.integrations.slack.message_builder import SlackBlock
+from sentry.integrations.slack.utils.escape import escape_slack_text
+from sentry.models.project import Project
+from sentry.models.release import Release
+from sentry.notifications.notifications.base import BaseNotification
+from sentry.services.hybrid_cloud.actor import RpcActor
+from sentry.tasks.summaries.utils import COMPARISON_PERIOD
+from sentry.types.integrations import ExternalProviders
+from sentry.utils import json
+
+from .base import SlackNotificationsMessageBuilder
+
+
+class SlackDailySummaryMessageBuilder(SlackNotificationsMessageBuilder):
+    def __init__(
+        self,
+        notification: BaseNotification,
+        context: Mapping[str, Any],
+        recipient: RpcActor,
+    ) -> None:
+        super().__init__(notification, context, recipient)
+        self.notification = notification
+        self.context = context
+        self.recipient = recipient
+
+    def linkify_error_title(self, group):
+        link = get_title_link(group, None, False, False, self.notification, ExternalProviders.SLACK)
```

Comment:
unrelated to your changes

any ideas on how we could improve `get_title_link` in the future? it's a bit hard to grok what's going on with these parameters and what each one is doing. it almost feels like we should have a `get_title_link` in the `slack.message_builder` and define a base `message_builder` in the BaseNotification?

Reply:
😬 I think this is an example of why Nathan is a big fan of using named params lol. If we at least give like `link_to_event` and `issue_details` a default value then we can avoid passing it and clean up the function calls.